apis/fluent-en_GB/atrium-api-fluent-en_GB/src/jvmTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/DateSubjectChangerSamples.kt

```diff
@@ -24,7 +25,7 @@ class DateSubjectChangerSamples {
                 .asLocalDate()                                  // subject is now of type LocalDate
                 .toBeAfter(LocalDate.parse("2025-07-17"))       // fails
                 .toBeBefore(LocalDate.parse("1996-07-17"))      // not evaluated/reported because `toBeAfter` already fails
-                                                                // use `.asLocalDate { ... }` if you want all assertions evaluated
+                                                                    // use `.asLocalDate { ... }` if you want all assertions evaluated
```

Comment:
please add the spaces after `//` because an auto-format of intellij will move the use again to the front